cmd/chisel/cmd_cut.go

```diff
@@ -24,12 +25,14 @@ var cutDescs = map[string]string{
 	"release": "Chisel release directory",
 	"root":    "Root for generated content",
 	"arch":    "Package architecture",
+	"timeout": "Timeout for each request",
```

Comment:
When we see `chisel cut --timeout 60s`, that clearly implies that the chisel command will timeout after 60s, but that's not what's document nor what it does. Maybe `--download-timeout`?

We also need to document how to provide the value. E.g.:
```
`Timeout for each download ("60s", or "3m", etc)`
```

Reply:
Depending on how the earlier proposal is accepted, this may be irrelevant.

Reply:
`--download-timeout` is more intuitive, thanks! I dropped the relevant changes anyway in https://github.com/canonical/chisel/pull/66/commits/06ee7b612b6be913d1eaad5acf26162f7480714d.